code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm

```diff
@@ -116,7 +117,7 @@
 
 /datum/action/xeno_action/activable/bombard/get_cooldown()
 	var/mob/living/carbon/xenomorph/boiler/X = owner
-	return X.xeno_caste.bomb_delay
+	return X.updated_delay
```

Comment:
this should use
`X.xeno_caste.bomb_delay - ((X.neuro_ammo + X.acid_ammo) * X.xeno_caste.ammo_multiplier)`
You wont need updated delay in that case

Reply:
Implemented